robot.py

```diff
@@ -64,8 +64,9 @@
 PISTON_RETRACT_ID = 5
 
 #hatch 
-RETRACT_ID = 6
-EXTEND_ID = 7
+EXTEND_ID = 6
```

Comment:
This is only 1/2 of the fix.     This would reverse the way that it functions.     6 is the "extend" solenoid, but it retracts the hatch mechanism, so that function needs to be reversed to preserve the functionality.   It works now, so you need to change the hatch operation so it still behaves the same.